Tests/State/LoadedStateSpecs.swift

```diff
@@ -120,16 +123,48 @@ final class LoadedStateSpecs: QuickSpec {
             }
         }
         
-        context("seek") {
+        context("seek completed") {
+            beforeEach {
+                // ARRANGE
+                self.mockPlayer.seekCompletionHandlerReturn = true
+                
+                // ACT
+                self.loadedState.seek(position: 42)
```

Comment:
La partie ACT est normalement à mettre dans chaque it, puisque c'est ce qui conditionne un comportement attendu. Après ce n'est pas grave en soi

Reply:
not really agree with you. it have to contain "assert" for me, and more if needed